src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -0,0 +1,170 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+import json
+import os
+import re
+import socket
+import urlparse
+
+from OpenSSL import crypto, SSL
+
+import sas
+import sas_testcase
+from util import winnforum_testcase
+
+
+_PEM_RE = re.compile(b"""-----BEGIN CERTIFICATE-----\r?
+.+?\r?
+-----END CERTIFICATE-----\r?\n?""", re.DOTALL)
+
+
+# Keeps this function as module function for easy mocking to adapt to specific
+# SAS implementation.
+def _VerifyCertificate(certificate_content, ca_bundle_content, is_server):
```

Comment:
I think it's better to leave it here if it won't be used in other test cases. Since we also have the "SSS" test cases (but no others that check security), we could consider a security_util.py? Otherwise util.py might become cluttered.